fdbserver/ptxn/TLogServer.actor.cpp

```diff
@@ -2233,7 +2295,7 @@ ACTOR Future<Void> updateStorageLoop(Reference<TLogGroupData> self) {
 }
 
 ACTOR Future<Void> tLog(
-    std::unordered_map<ptxn::TLogGroupID, std::pair<IKeyValueStore*, IDiskQueue*>> persistentDataAndQueues,
+    std::unordered_map<ptxn::TLogGroupID, std::pair<IKeyValueStore*, IDiskQueue*>> oldPersistentDataAndQueues,
```

Comment:
The original name without `old` seems more accurate. The data store and disk queue will also be used in the new tlog, aren't they?

Reply:
no, each group has a new pair of `IKeyValueStore` and `IDiskQueue`, the ones for the new group is in the request.

Reply:
here is the new pair for the new group https://github.com/apple/foundationdb/pull/6210/files#diff-aa6ef9ddb78f76c19d1c39dd37cb11408998cd9b20ef04ab8de49c453e04c577R102

Reply:
If a TLog restarts, and being re-recruited as a TLog, will it create new files? For the current implementation in `master`, it uses the existing files.

Reply:
[here is about reading old files, and `restoreFromDisk` is true](https://github.com/apple/foundationdb/blob/main/fdbserver/worker.actor.cpp#L1574).

[here is about creating new files for a new generation, look at how the `queue` is created with a newly generated `logId`](https://github.com/apple/foundationdb/blob/main/fdbserver/worker.actor.cpp#L1928)